components/sceneutil/mwshadowtechnique.cpp

```diff
@@ -553,7 +553,7 @@ MWShadowTechnique::ShadowData::ShadowData(MWShadowTechnique::ViewDependentData*
     // set up the camera
     _camera = new osg::Camera;
     _camera->setName("ShadowCamera");
-    _camera->setReferenceFrame(osg::Camera::ABSOLUTE_RF_INHERIT_VIEWPOINT);
+    _camera->setReferenceFrame(osg::Camera::ABSOLUTE_RF);
```

Comment:
This will break terrain shadows with distant terrain enabled - the shadow cameras are often a few miles above Vvardenfell, so the terrain system would choose the lowest quality LOD level for those cameras and that's not enough detail to get terrain shadows from anything other than Red Mountain during late evening. To get correct terrain shadows with working self-shadowing, the same LOD nodes need to be used for the shadow cameras and main camera, and that means we need to inherit the viewpoint.

Reply:
my excuses i thought it was an error..i rollback this hacky change

Reply:
arf not so anodine in fact